File: Selenium/Activities/Activity6.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.common.action_chains import ActionChains


service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://training-support.net/selenium/input-events")
    print("The title of the page is: ", driver.title)
    #Press the key of first letter of your name in caps
    #Press CTRL+a and the CTRL+c to copy all the text on the page.
    actions = ActionChains(driver)
    actions.send_keys("M").key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(Keys.CONTROL).perform()
    

File: Selenium/Activities/activity6.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class activity6 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        Actions builder = new Actions(driver);
        driver.get("https://www.training-support.net/selenium/input-events");
        System.out.println("The Title of the page is: " + driver.getTitle());
        //Press the key of first letter of your name in caps
        builder.sendKeys("M").build().perform();
        //Press CTRL+a and the CTRL+c to copy all the text on the page.
        builder.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("C").keyUp(Keys.CONTROL).build().perform();
        driver.quit();

    }
}
